### PR TITLE
keep changes to package.json and remove exit.tsx

### PR DESCRIPTION
**Pull request by ManavDevF**
add git huskey along with prettier and ts lint

### CODE REVIEW

File: main
Comment (vaibhavsrivastava7777) [inline main:1]: why this change?

Comment (ManavDevF) [reply main:1]: file not needed .
